src/index.ts

```diff
@@ -75,6 +75,20 @@ export function stubInterface<T extends object>(methods: ObjectMethodsMap<T> = {
     })
 }
 
+function getObjectMethods(object: object): Array<string> {
+    let methods: Array<string> = [];
```

Comment:
nitpick; can be `const` since you aren't reassigning `methods`

Reply:
You're right, don't know why I used here `let`